sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_boto3.ipynb

```diff
@@ -0,0 +1,1541 @@
+    "\n",
+    "* Measure biases that can occur during each stage of the ML lifecycle (data collection, model training and tuning, and monitoring of ML models deployed for inference).\n",
+    "* Generate model governance reports targeting risk and compliance teams and external regulators.\n",
+    "* Provide explanations of the data, models, and monitoring used to assess predictions.\n",
+    "\n",
+    "This sample notebook walks you through:  \n",
+    "1. Key terms and concepts needed to understand SageMaker Clarify\n",
+    "1. Measuring the pre-training bias of a dataset and post-training bias of a model\n",
+    "1. Explaining the importance of the various input features on the model's decision\n",
+    "1. Accessing the reports through SageMaker Studio if you have an instance set up.\n",
+    "\n",
+    "In doing so, the notebook first trains a [SageMaker XGBoost](https://docs.aws.amazon.com/sagemaker/latest/dg/xgboost.html) model using training dataset, then utilizes the [AWS SDK for Python](https://aws.amazon.com/sdk-for-python/) to launch SageMaker Clarify jobs to analyze a testing dataset in CSV format. Additionally, there are peer examples available that use the [SageMaker Python SDK](https://sagemaker.readthedocs.io/en/stable/) to launch SageMaker Clarify jobs to analyze data [in CSV format](https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb) and [in JSON Lines format](https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker_processing/fairness_and_explainability/fairness_and_explainability_jsonlines_format.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites and Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import boto3\n",
+    "import time\n",
+    "from datetime import datetime\n",
+    "from sagemaker import get_execution_role, session"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set configurations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Region: ap-south-1\n",
+      "Role: arn:aws:iam::003394947794:role/service-role/SMClarifySageMaker-ExecutionRole\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Initialize sagemaker session\n",
+    "sagemaker_session = session.Session(boto3.Session())\n",
```

Comment:
No need to create a new boto3 session, `sagemaker_session = session.Session()` should work.